src/backend/vulkan/src/lib.rs

```diff
@@ -394,18 +419,54 @@ impl CommandQueue {
     }
 }
 
-impl core::CommandQueue<Backend> for CommandQueue {
-    unsafe fn submit_raw<'a, I>(
-        &mut self,
-        submit_infos: I,
+impl core::RawCommandQueue<Backend> for CommandQueue {
+    unsafe fn submit_raw(&mut self,
+        submission: core::RawSubmission<Backend>,
```

Comment:
Why only single submission per submit?

Reply:
Looking at it again, I think we should get it back to multiple submissions per call, just to match Vulkan if nothing else...